cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -44,9 +45,9 @@ const (
 
 // CertManagerClient has methods to work with cert-manager components in the cluster.
 type CertManagerClient interface {
-	// EnsureWebhook makes sure the cert-manager Webhook is Available in a cluster:
+	// EnsureCertManager makes sure the cert-manager Webhook is Available in a cluster:
```

Comment:
Maybe we update the entire comment since the Webhook here was referring to the API service webhook.